.buildkite/hooks/pre-exit

```diff
@@ -10,7 +10,7 @@ if [[ "$BUILDKITE_PIPELINE_SLUG" == "elastic-agent" && "$BUILDKITE_STEP_KEY" ==
 
   # Perform cleanup of integration tests resources
   echo "--- Cleaning up integration test resources"
-  TEST_INTEG_AUTH_ESS_REGION=azure-eastus2 SNAPSHOT=true mage integration:clean
+  TEST_INTEG_AUTH_ESS_REGION=aws-eu-central-1 SNAPSHOT=true mage integration:clean
```

Comment:
I originally tested with us-east-1 but it seems this works as well (I just created a deployment manually to check).

Only thing is if our buildkite instances are in NA then this will be cross-continental traffic in the tests.

Reply:
> Only thing is if our buildkite instances are in NA then this will be cross-continental traffic in the tests. 

Ooof, yeah, good point. Let me switch it to `us-east-1`.

Reply:
Changed in 37e5933f4197dc90a8690a8c207366d2ddb26bbe.